config/config.go

```diff
@@ -40,12 +40,13 @@ func LoadConfig(configFile string) Config {
 	return cfg
 }
 
-func (cfg Config) GetEnv(env string) EnvDirSection {
+func (cfg Config) GetEnv(env string) (EnvDirSection, error) {
 	if conf, ok := cfg.Sections[env]; ok {
 		conf.ParsePerms()
-		return *conf
+		return *conf, nil
```

Comment:
What do you think of changing the return type to `(* EnvDirSection, error)`? Then you wouldn't need to to return an empty `EnvDirSection` on line 49